src/shared/ui/CustomButton/CustomButton.jsx

```diff
@@ -0,0 +1,5 @@
+import "./CustomButton.css";
+
+export const CustomButton = ({ label, ...sxStyles }) => {
```

Comment:
as a test component, it's okay, but it doesn't make sense because it does nothing, we just paste the props (btw instead of `label` probable should be `children`) but for now keep it as is

Reply:
aha , ok